src/app/(app)/(tabs)/profile.tsx

```diff
@@ -25,104 +68,113 @@ export default observer(function ProfileScreen() {
+        />
+        <Text
+          preset="formLabel"
+          tx="demoProfileScreen.rnFamiliarity"
+          style={{ marginBottom: spacing.xs }}
+        />
+        <Text
+          tx={`demoProfileScreen.familiaritySubtitles.${rnFamiliarity}` as TxKeyPath}
+          style={$familiaritySubtitle}
+        />
+        <Slider
+          minimumValue={0}
+          maximumValue={4}
+          minimumTrackTintColor={colors.tint}
+          maximumTrackTintColor={colors.palette.secondary500}
+          tapToSeek
+          step={1}
+          value={rnFamiliarity}
+          onValueChange={(value) => setProp("rnFamiliarity", value)}
+          style={$slider}
+          renderStepNumber
+          StepMarker={({ stepMarked }) => (
+            <View
+              style={[
+                $stepMarkerStyle,
+                stepMarked && {
+                  backgroundColor: colors.transparent,
+                },
+              ]}
+            />
+          )}
+        />
 
-      <Toggle
-        labelTx="demoProfileScreen.job"
-        variant="switch"
-        labelPosition="left"
-        containerStyle={$textField}
-        value={openToWork}
-        onPress={() => setProp("openToWork", !openToWork)}
-      />
-      <Toggle
-        labelTx="demoProfileScreen.remote"
-        variant="switch"
-        labelPosition="left"
-        containerStyle={$textField}
-        value={remote}
-        onPress={() => setProp("remote", !remote)}
-      />
-      <Text preset="formLabel" tx="demoProfileScreen.skills" />
-      <TextField
-        value={skills}
-        containerStyle={$textField}
-        placeholderTx="demoProfileScreen.skills"
-        onChangeText={(text) => setProp("skills", text)}
-      />
-      <TextField
-        labelTx="demoProfileScreen.bio"
-        containerStyle={$textField}
-        multiline
-        placeholderTx="demoProfileScreen.bio"
-        value={bio}
-        onChangeText={(text) => setProp("bio", text)}
-      />
-      <Button
-        tx="demoProfileScreen.submitButton"
-        preset="filled"
-        onPress={() => console.log("Validaiton done. Submitting to API.")}
-      />
-    </Screen>
+        <Toggle
+          labelTx="demoProfileScreen.job"
+          variant="switch"
+          labelPosition="left"
+          containerStyle={$textField}
+          value={openToWork}
+          onPress={() => setProp("openToWork", !openToWork)}
+        />
+        <Toggle
+          labelTx="demoProfileScreen.remote"
+          variant="switch"
+          labelPosition="left"
+          containerStyle={$textField}
+          value={remote}
+          onPress={() => setProp("remote", !remote)}
+        />
+        <SelectField
+          options={skillsList}
+          labelTx="demoProfileScreen.skills"
+          onSelect={(selected) => setProp("skills", selected)}
+          value={skills}
+          renderValue={(value) => t("demoProfileScreen.skillsSelected", { count: value.length })}
```

Comment:
I think `t` can be `translate` so it comes from `src/i18n/` - this way we get the type checking on the txKey.

Which lead me to find that we're missing this txKey - so we can either add it to `main` or make it part of the steps somewhere in this lesson